test/simulator/timestamp/call_log_manager/call_log_manager.cpp

```diff
@@ -51,6 +51,7 @@ call_log_manager::api_map_setup()
     _api_map["close_session"] = api_method::close_session;
     _api_map["open_session"] = api_method::open_session;
     _api_map["set_timestamp"] = api_method::set_timestamp;
+    _api_map["query_timestamp"] = api_method::query_timestamp;
```

Comment:
We could sorth them A-Z

Reply:
Updated in [4a62e95](https://github.com/wiredtiger/wiredtiger/pull/8145/commits/4a62e95768f31c15421f190f70c93abf28157f99)